### DISCUSSION TITLE
Automatically split width scan into multiple pages

### DISCUSSION THREAD

**Discussion by theopoon** [gotson/komga#1704] [Ideas]
Due to how Japanese Manga were drawn and scanned (compare to US comics), two pages were often stored as one image, example:
image004.ext contains page05 & page06

This does not seems much issue on a desktop / landscape client, but it is very hard to navigate and read on a mobile device that is portrait.

In the early PDA days most manga reader do have ability to "auto split image wider than screen dimension" into 2 or "force split of landscape image into portrait / 2"

For example, an application called MangaMeeya do have such function (auto single page split, left & right / top & bottom) and if also feature left or right first as option.

That would be great to have function for api / web viewer.

**Comment by menkinds** [comment]
Very helpful

**Comment by Tuxrug** [comment]
Branching off on this idea, being able to rotate a page and mark it as a double would be great. Probably too much of a fringe case to be worth building-in, but I obtained some comics where double-page spreads were stored as a 90-degree-rotated single page. For a few I've manually unzipped the CBZs, rotated and split the pages, re-zipped the CBZ and re-analyzed it, but I've got enough that a built-in feature to compensate for that issue would make it much less of a headache.
